website_event_filter_selector/tests/test_ui.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo.tests.common import HttpCase
+from datetime import datetime, timedelta
+
+
+class UICase(HttpCase):
+    def test_ui_website(self):
+        """Test frontend tour."""
+        templates = {
+            "website_event": {
+                "event_left_column",
+                "event_category",
+                "event_location",
+            },
+            "website_event_filter_selector": {
+                "city_left_column",
+                "filter_city",
+                "filter_country",
+                "filter_date",
+                "filter_type",
+                "user_custom",
+            },
+        }
+        with self.cursor() as cr:
+            env = self.env(cr)
```

Comment:
This hack is no longer needed in v12. Remove the `with` block and use `self.env` instead of `env`.